lib/ansible/modules/cloud/digital_ocean/do_sshkeys.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
```

Comment:
@ryansb keep in mind that the API is slightly different.

Changes done:
* Rename to ** digital_ocean_sshkeys**
* Remove deprecation of old module (since we kept the name this doesn't make sense)